epub33/core/index.html

```diff
@@ -9721,8 +9721,16 @@ <h3>The <code>epub:type</code> attribute</h3>
 
 					<dt>Usage:</dt>
 					<dd>
-						<p><a data-cite="html#global-attributes">Global attribute</a>. [=EPUB creators=] MAY specify on
-							all elements.</p>
+						<p><a data-cite="html#global-attributes">Global attribute</a>. It MAY be used
+							in [[html]], [[svg]], and on [=Media overlay documents=] with the following restrictions:</p>
+						<ul>
+							<li>For [[html]] content: it MUST be used on [=palpable content=].</li>
+							<li>For [[svg]] content: it MUST be used on 
+								<a href="https://www.w3.org/TR/SVG11/intro.html#TermStructuralElement">structural</a>,
+								<a href="https://www.w3.org/TR/SVG11/intro.html#TermShapeElement">shape</a>, or
+								<a href="https://www.w3.org/TR/SVG11/text.html">text</a> elements.</li>
+							<li>For [=media overlay document=]: it MUST be used on the [^body^] element or its descendants.</li>
```

Comment:
The MO content model defines where epub:type can be used, so I don't think we need a bullet.

Maybe this list could just be for HTML and SVG and then have a paragraph or note after that says "For use in [=media overlay documents=] refer to `<a href="#sec-overlays-def"/>.`"

Reply:
Agreed. Will modify this accordingly.